package.json

```diff
@@ -41,7 +41,9 @@
     "@react-navigation/native-stack": "^6.9.5",
     "axios": "^1.2.1",
     "expo": "~47.0.8",
+    "expo-build-properties": "~0.4.1",
```

Comment:
Do we need this? Isn't this for expo prebuild: https://docs.expo.dev/versions/latest/sdk/build-properties/

Reply:
Yeah it's needed for native dependencies, I couldn't get it to build for iOS without this.
https://stackoverflow.com/questions/73415961/getting-error-when-building-ios-app-by-using-eas-expo-application-services

Reply:
Makes sense 👍